### PR TITLE
Add TEAL source in read-only editor

### PR DESCRIPTION
**Pull request by barnjamin**
debugging why second ace editor doesn't show any content still

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,17 @@ it can currently deploy stateful apps as well, but there is no UI feature to int
 
 ### tldr
 
-clone this repo, go into the `src/server` directory, then `pip install -r requirements.txt`. you can locally start your server by `flask run`, and you'll see the editor on localhost port 5000. in order to compile and deploy apps, you need to have [sandbox](https://github.com/algorand/sandbox) running. 
+Make sure you have [sandbox](https://github.com/algorand/sandbox) running.
```
Comment (algochoi) [inline README.md:18]: thanks for cleaning this up 😄  I think you can delete the `### tldr` above this line now

File: src/server/static/js/editor.js [hunk 2]
```diff
@@ -28,17 +30,71 @@ def approval():
...
+`
+
 const consoleMessages = [];
 
 const editorLib = {
     init() {
-        editor.session.setMode("ace/mode/python");
+        editor.getSession().setMode("ace/mode/python");
+
+        //TODO: can we define our own mode? formatting? syntax highlighting?
```
Comment (algochoi) [inline src/server/static/js/editor.js:86]: I think this is a good idea - if we also use monaco, we might be able to port over the existing TEAL syntax highlighting rules in VSCode like the one [here](https://github.com/ObsidianLabs/vscode-algorand/blob/master/syntaxes/teal.tmLanguage.json)

**Review by algochoi — APPROVED** [review]
Really cool feature, I love it - feel free to merge in whenever 🎉

File: src/server/static/js/editor.js (hunk 2 above)
Comment (barnjamin) [reply src/server/static/js/editor.js:86]: https://github.com/ajaxorg/ace/wiki/Importing-.tmtheme-and-.tmlanguage-Files-into-Ace looks like theres an example of importing that for ace

Comment (barnjamin) [reply src/server/static/js/editor.js]: Got this to work but punted on importing and just copy/pasta'd the resulting ace define stuff into editor.js with some tweaks to the way it gets defined.

**Pull request merged**